src/ContentCard/index.tsx

```diff
@@ -34,7 +34,7 @@ interface ContentCardProps {
    * Amount of border radius to add on all sides of card.
    */
   radiusSize?: "s" | "m" | "l";
-  onClick?: () => void;
+  onClick?: (e?: React.MouseEvent) => void;
```

Comment:
We do NOT pass an event object when calling the passed onClick function when handling key events, but it looks like the event is passed on mouse click events.

Reply:
I like that.